department-of-veterans-affairs/vets-website
# MHV-59185 June-Staging-Review-Finding-Correct-The-Breakpoint-And-Column-Structure-For-Side-Nav

## Summary
I replaced `small-screen:vads-u-flex-direction--row` with `medium-screen:vads-u-flex-direction--row`. This change resolved the issue of the side navigation collapsing incorrectly at the 481 breakpoint. After the fix, the side navigation now appears as a single column at this breakpoint.

## Related issue(s)

[MHV-59185](https://jira.devops.va.gov/browse/MHV-59185) - June Staging Review finding: Correct the breakpoint and column structure for side nav

June Staging Review finding: Correct the breakpoint and column structure for side nav

Details found in Github:

86221-Staging Review finding: Correct the breakpoint and column structure for side nav
https://github.com/department-of-veterans-affairs/va.gov-team/issues/86221

Description
The side nav collapses at the 481 break point which is incorrect and it is still in 2 columns. Then at 480 it shifts to the correct 1 column layout.

Link, screenshot or steps to recreate

Recommended action
At the [768 break point](https://design.va.gov/foundation/breakpoints#names-and-values) the [side nav](https://design.va.gov/components/sidenav) should collapse and become 1 column. See about [VA health benefits](https://www.va.gov/health-care/about-va-health-benefits/) for an example. Make sure you have the correct code and update the breakpoint and column structure.

User Story: As a Medical Records User, I want to see the side navigation collapse at the appropriate times.

 

 

Feature Flag Y/N ? N

DataDog/Analytics Y/N ? N

Manual Testing Y/N ?  Y-Maruf

Automated Testing Y/N ? N

Accessibility Testing Y/N ? N

UCD Validation Y/N N

 

Acceptance Criteria:

AC1  Side nav and column structure have been corrected at the 481 break point

AC2 Close the GH ticket when resolved

AC3
AC4
AC5

 

Links to UCD:

Figma Link:

User Flow:

Mobile:

Desktop:

Other Design Notes:

Architectural Notes: Always check for the latest designs in Figma

Test Coverage
No Tests were found testing the requirement.

## Testing done
Conducted UI testing to ensure the expected behavior. I updated existing unit test for new components.

## Screenshots
![Screenshot 2024-07-01 at 10 48 54 AM (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/8038703/28c96ebf-e472-4a31-944c-266487736a75)

## What areas of the site does it impact?

The modification affects Nav section of Medical Records pages.

## Acceptance criteria
The navigation collapses into a single column at a screen width of 841 pixels.

### Quality Assurance & Testing


### Authentication
